tests/components/alexa/test_smart_home.py

```diff
@@ -273,6 +274,79 @@ def assert_endpoint_capabilities(endpoint, *interfaces):
     # tests
 
 
+async def test_effect_light(hass):
+    """Test effect light discovery."""
+    device = (
+        "light.test_4",
+        "on",
+        {
+            "friendly_name": "Test light 4",
+            "supported_features": 4,
+            "effect_list": [0, 1, 2, 3, 4],
```

Comment:
```suggestion
            "effect_list": [0, 1, 2, 3, 4, "colorloop", "color_loop", "random", "white", "0", "1","Rainbow","In In","Sparkle+"],
```
Test for other potential values, including int as strings, spaces in name, etc.